samples/keda-sqs/specs/README.md

```diff
@@ -0,0 +1,32 @@
+# Spec commands
+
+Before you create the specs, please follow the setup steps in the [README](../README.md).
+
+## Spec Generation
+
+```sh
+fission spec init
+fission env create --name go --image fission/go-env-1.14 --builder fission/go-builder-1.14 --spec
+fission pkg create --name producer --env go --src producer/producer.zip --spec
```

Comment:
This can be 
```suggestion
fission pkg create --name producer --env go --src "producer/*" --spec
```
So you don't have to zip

Reply:
Oh and also - you don't need to explicitly create package too. You can use the within the function create command this: `--src "producer/*"` and it will create package!